azure-kit-demo/README.md

```diff
@@ -0,0 +1,55 @@
+# Azure Kit Demo
+
+## Run the demo
+
+To run the demo you first need to have a local Kubernetes cluster up and running.
+You can find recommended tools here: https://kubernetes.io/docs/tasks/tools/
+
+To deploy the demo application into your cluster, follow these steps:
+
+1. Build the application JAR:
+```
+# mvn clean package -Pproduction -Dmaven.javadoc.skip=true -Dmaven.source.skip=true
```

Comment:
System properties can now be removed, since they are configured in POM file
```suggestion
# mvn clean package -Pproduction
```

Reply:
Done.